### ISSUE TITLE
关于test_simple_SQL_config.py可视化

### ISSUE THREAD

**Issue by chaoying0115** [Lavreniuk/SPIdepth#12]
您好，非常感谢您的出色工作。我用test_simple_SQL_config.py进行图片测试，得到了以下图片，与论文风格有些不一致，请问是哪里出了问题？
![SQL_512_disp](https://github.com/user-attachments/assets/796c3323-c72c-473a-abb6-c3f77f3fcc06)

**Comment by Lavreniuk** [comment]
@chaoying0115 , thank you for interesting in our model, have you set checkpoints path?
please find attached image and result for which I just double checke
![02](https://github.com/user-attachments/assets/bee15477-6ab6-4e77-8380-3bde6c2b5397)
![02](https://github.com/user-attachments/assets/3a19340f-11d2-4ce5-b64b-c0eee1133907)
d.

**Comment by chaoying0115** [comment]
您好 非常感谢您的回复。我是SQLdepth那边下载的模型进行测试，但测出的情况如下
/
![640](https://github.com/user-attachments/assets/c85857ce-e713-4f19-9d9c-e9cd516f2510)
![640](https://github.com/user-attachments/assets/77bf9bf3-fd19-40e6-a5a6-0d4a1d75b4a6)

这是代码的运行情况
![image](https://github.com/user-attachments/assets/f1fc996e-82b4-4c03-b005-a5866ad147f5)
![image](https://github.com/user-attachments/assets/34bb13e6-b4e1-47fe-9ba3-932d78f3ee71)

不知道问题出在哪里 期待得到您的回复！

**Comment by Lavreniuk** [comment]
for locating the problem, could you run the inference on the image that I provided? 
if you will get same result as I on that image, it means everything work, if not - so something wrong with your set up.

**Comment by chaoying0115** [comment]
您好 我是用您提供的图像进行推理 得到以下结果
![test](https://github.com/user-attachments/assets/c7a47143-b11d-4858-bc41-6786e2584359)
这是运行截图和相关文件设置
![image](https://github.com/user-attachments/assets/d87baca5-64b7-402e-9c49-5f723c49265d)
[args_test_simple_kitti_320x1024.txt](https://github.com/user-attachments/files/17843142/args_test_simple_kitti_320x1024.txt)
我使用的是SOLdepth 提供的权重 https://drive.google.com/file/d/1_BHfGXqVsE4RrCM58_5nEzQHJmyxDsQO/view

**Comment by Lavreniuk** [comment]
ok, but as I saw you used other config than I provided in the readme file.
Please use this one and just download the pretrained weights and change the path for them.
conf/cvnXt.txt

**Comment by chaoying0115** [comment]
好的 谢谢您。我按照您的文件，下载了https://drive.google.com/file/d/14Hb8UsjraLWk1TvtMPd5moEDMlDxiHkC/view 的权重
文件设置和运行结果如下，似乎出现了权重不匹配的问题 是我权重下载错了吗？
![image](https://github.com/user-attachments/assets/04135632-0f7e-4ef2-ba54-dff486b35655)
![image](https://github.com/user-attachments/assets/563dbf8a-44e2-4d94-a387-150fd17fc9fe)
![image](https://github.com/user-attachments/assets/70503bc8-ff15-43c3-9f30-feb64176feb1)

**Comment by Lavreniuk** [comment]
I think you have downloaded some other model.
Please double check that you have downloaded from the link that I provided in readme:
https://huggingface.co/MykolaL/SPIdepth/tree/main/kitti

**Comment by chaoying0115** [comment]
谢谢 在您指导下我得到与您一样的可视化。我发现我先前的错误是发生在自监督Resnet-50预训练模型的加载上，他们似乎不会去加载文件提供相关权重文件，而您提供的是全监督情况下的可视化所以没有问题
![image](https://github.com/user-attachments/assets/2deebf44-c98a-48da-8ce7-106fd2ef75c

您有尝试做过自监督模型的可视化吗？能够得到正确风格的可视化图像？

**Comment by chaoying0115** [comment]
谢谢 在您指导下我得到与您一样的可视化。我发现我先前的错误是发生在自监督Resnet-50预训练模型的加载上，他们似乎不会去加载文件提供相关权重文件，如图所示，而您提供的是全监督情况下的，可以加载文件提供的权重，所以没有问题

![image](https://github.com/user-attachments/assets/a471d7dc-6b0d-4526-a498-6027210c5d1e)

您有尝试做过自监督模型的可视化吗？能够得到正确风格的可视化图像？

**Comment by Lavreniuk** [comment]
yes, you could do same for unsupervised model, but you need to train it by yourself, because I provided only final weights.
